website/templates/project/statistics.mako

```diff
@@ -5,6 +5,16 @@
   <h2 class="text-300">Analytics</h2>
 </div>
 
+<div id="hiddenFlag" class="ad_block_display">
+  <div class="m-b-md p-md osf-box-lt box-round text-center">The use of adblocking software may prevent site analytics from loading properly.</div>
+</div>
+<script src="/static/js/ads.js"></script>
+<script>
+  if( window.canRunAds === undefined ){
+    document.getElementById('hiddenFlag').style.display = "block";
```

Comment:
Can I ask what the link goes to? It just keeps going to this file for me and do you mean line 14? Are these two different things you are saying I should do or is this all one thing?

Also I am familiar with jquery but I was sure you still needed to put it into a script tag in order for it to work. I do not think jquery can be run in html like you are suggesting.